sw/airborne/firmwares/rotorcraft/stabilization/stabilization_rate_indi.c

```diff
@@ -127,33 +129,37 @@ void stabilization_rate_set_setpoint_i(struct Int32Rates *pqr)
   RATES_FLOAT_OF_BFP(stabilization_rate_sp, *pqr);
 }
 
-static void stabilization_indi_rate_calc_cmd(int32_t indi_commands[], struct FloatRates rate_ref, bool in_flight)
+static void stabilization_indi_rate_calc_cmd(int32_t indi_commands[], bool in_flight)
 {
   /* Get the angular acceleration references */
   indi.angular_accel_ref.p =  indi.reference_acceleration.rate_p * (rate_ref.p - body_rates->p);
   indi.angular_accel_ref.q =  indi.reference_acceleration.rate_q * (rate_ref.q - body_rates->q);
   indi.angular_accel_ref.r =  indi.reference_acceleration.rate_r * (rate_ref.r - body_rates->r);
 
+// WE NEED TO CHOOSE BETWEEN STABILIZATION_INDI_SIMPLE AND STABILIZATION_INDI - NOT SURE IF I DID CORRECTLY HERE:
```

Comment:
Could someone tell me if I did correctly here? Because we have two INDI files. One, which is the `_simple` and second which considers a control allocation method.

Reply:
The code in this function has several errors I think.
Beside that, even if the prototype of the `calc_cmd` function is the same (rate ref as input, command array as output), it will still be necessary to use an `ifdef` as array outputs are different.